contacts/contacts.service.js

```diff
@@ -0,0 +1,144 @@
+import {
+  data
+} from './data';
+import {
+  Person
+} from './entities';
+import {
+  CONTACTS_SERVICE_OBJECT_NAME,
+  NOT_FOUND,
+  MESSAGES
+} from './constants';
+import {
+  sortArrayAlphabetically,
+  sortArrayByLikes,
+  isUndefined,
+  isZero,
+  isNull,
+  createPersonInstance,
+  localStorageServie
+} from './helper';
+
+
+export class ContactsService {
+
+  constructor() {
+    this.internalArrayContacts = [];
+    this.init();
+  }
+
+  init = () => {
+    const localStorageObject = localStorageServie.getFromLocalStorage(CONTACTS_SERVICE_OBJECT_NAME);
+    if (!isNull(localStorageObject) && !isZero(localStorageObject.length)) {
```

Comment:
you should be able to write it like this

this.internalArrayContacts = localStorageObject ? localStorageObject.map(item => createPersonInstance(item)) : data.map(item => createPersonInstance(item));